### ISSUE TITLE
Segmented Controls Accessibility section missing

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus.trimble.com#466]
There is no content here:
https://modus.trimble.com/components/mobile/segmented-controls/accessibility/

Should we hide the Accessibility tab from this section so the page is inaccessible?
